src/librustc_resolve/error_codes.rs

```diff
@@ -1660,6 +1660,20 @@ fn const_id<T, const N: T>() -> T { // error: const parameter
 }
 ```
 "##,
+
+E0734: r##"
```

Comment:
You'll have to make it E0735, I created E0734 just a few days ago.

Reply:
thanks for the heads up